app/views/application/_navbar.html.haml

```diff
@@ -1,11 +1,15 @@
 -# locals: menu_nodes, active
 
 %aside.sidebar
-  %nav.primary-nav{'aria-label' => 'main navigation'}
+  %nav.local-nav{'aria-label' => 'main navigation'}
     %a#primary-nav-heading.js-accordion-trigger.nav-heading{href: 'javascript:void(0)',
             'aria-expanded' => true, 'aria-controls' => 'primary-nav-panel'}
```

Comment:
Potentially - I intentionally left the JS stuff as I'm not sure what their implementation is relying on. I'll make the change because it makes sense to, and consult with the uikit folk.